docs-partials/builder/vmware/RemoteHypervisorBuilds.mdx

```diff
@@ -0,0 +1,44 @@
+## Building on VMware vSphere Hypervisor
+
+In addition to using the desktop virtualization products to build virtual
+machines, this plugin can use a VMware vSphere Hypervisor to build the virtual
+machine.
+
+Before using a vSphere Hypervisor, you need to enable `GuestIPHack` by
+running the following command:
+
+```shell-session
+$ esxcli system settings advanced set -o /Net/GuestIPHack -i 1
+```
+
+When using the vSphere Hypervisor, the builder still downloads the ISO and
+various files locally, and uploads these to the remote hypervisore. This plugin
+uses SSH to communicate to the ESXi host rather than the vSphere API. [^2]
+
+This plugin also requires VNC to issue boot commands during a build. Please
+refer to the VMware vSphere documentation on how to update the hypervisor's
+firewall to allow these connections. VNC can be disabled by not setting a
+`boot_command` and setting `disable_vnc` to `true`.
+
+Please note that you should disable vMotion for the host you intend to run
+builds on; a vMotion event will cause the build to fail.
+
+To run a remote build for your virtual machine image, use the following
+configurations:
+
+**Required**:
+
+- `remote_type`
```

Comment:
This would be very repetitive to include the full content again. I shortened to just the configuration needed and removed the duplications (content and content insertion) from each MDX. Given that ESXi is marked as deprecated it might be best to retain this "as-reduced" so it's less to remove in the future.

Is there a way to have these link back to the configuration option origination.

Reply:
Hmmm, you could maybe add a link with a fragment that points to a section of the documentation? We have a couple of such instances in plugins, like on the [AWS](https://github.com/hashicorp/packer-plugin-amazon/blob/main/builder/ebs/builder.go#L63) plugin for example. You'd need to know the name of the fragment to point to, and that might do it.
Given this is markdown, I can also envision adding a random span with an anchor somewhere in the docs, and using that as the link fragment. But I don't know that it's been done before so take that with a grain of salt :)

Reply:
Addressed in https://github.com/hashicorp/packer-plugin-vmware/commit/5d329c5d54ee0bbb6ff4198a44225494bb1e76fc. Ready for review again.